### DISCUSSION TITLE
Offline media player

### DISCUSSION THREAD

**Discussion by ghost** [polymorphicshade/NewPipe#245] [Q&A] [answered]
Hi since official newpipe dev won't provide offline media player to newpipe
If you don't mind can you add it to your newpipe sponsoblock fork?
Or do you have any offline media player that have ui like YouTube?

Thanks 🙏

**Comment by polymorphicshade** [comment, marked as answer]
> Hi since official newpipe dev won't provide offline media player to newpipe
> If you don't mind can you add it to your newpipe sponsoblock fork?
> Or do you have any offline media player that have ui like YouTube?
> 
> Thanks 🙏

https://github.com/polymorphicshade/NewPipe/issues/30
